front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx

```diff
@@ -49,7 +49,7 @@ const STDCMAllowances = () => {
   };
 
   return (
-    <div className="row osrd-config-item mb-2 osrd-config-item-container">
+    <div className="d-flex mb-2 align-items-center osrd-config-item-container">
```

Comment:
I'm not sure align-items-center is useful here, I removed it and it didn't change anything. What do you think ?
```suggestion
    <div className="d-flex mb-2 osrd-config-item-container">
```